lib/ffmpeg/encoderbase.py

```diff
@@ -47,6 +47,7 @@ def gen_output_opts(self):
 
     if vars(self).get("profile", None) is not None:
       opts += " -profile:v {mprofile}"
+      opts += self.gen_hevc_main10sp_opts()
```

Comment:
This function is not defined in base class or other subclasses.  Thus, it will break vaapi, dxva2 and d3d11 encode subclasses.  

How about we return `"main10 -main10sp 1"` from the ffmpeg/qsv/util.py::mapprofile(...) function?  Then `self.mprofile` will already include it and we can remove this new function.

Reply:
sure, thanks Artie

Reply:
updated